disco_aws_automation/disco_deploy.py

```diff
@@ -481,3 +488,26 @@ def update(self, dry_run=False):
         amis = self.get_update_amis()
         if len(amis):
             self.update_ami(random.choice(amis), dry_run)
+
+    def hostclass_option(self, hostclass, key):
```

Comment:
yes, there should be asiaq specific instance of parsed options or config parser which will encapsulate this logic